reactioncommerce/catalyst
# fix: DataTable QoL and fixes

Resolves #137 
Impact: **minor**  
Type: **feature|performance**

## Summary

- Debounce global filter
- Add a `refetch()` function
- Add a `fetch()` function

## Testing
1. Ensure data table works with fetch and refetch
1. See that the global filter performs more quickly

Refetch happens with the bulk actions, test fetch with the `Custom controls` below the table
